lib/matplotlib/tests/test_axes.py

```diff
@@ -797,6 +797,23 @@ def test_polar_rlim_zero():
     assert ax.get_ylim()[0] == 0
 
 
+@image_comparison(['aitoff_proj'], extensions=["png"], remove_text=True)
+def test_aitoff_proj():
+    """
+    Test aitoff projection ref.:
+    https://github.com/matplotlib/matplotlib/pull/14451
+    """
+    np.random.seed(42)
+    ra_random = np.random.rand(20) * 2 * np.pi
```

Comment:
Can these be a structured grid (or line of points) instead of random points? That way it's easier to understand changes to the positioning of the points due to any future code changes.

Reply:
Ok, changed to use a grid. See updated post above.